src/frontend/use-replicache.ts

```diff
@@ -48,12 +48,12 @@ export function useReplicache<M extends MutatorDefs>({
     // - https://doc.replicache.dev/how-it-works#poke-optional
     // - https://github.com/supabase/realtime
     // - https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events
-    const cancelReceiver = getPokeReceiver()(name, async () => r.pull());
+    const cancelReceiver = getPokeReceiver()(name, () => r.pull());
```

Comment:
There is a slight difference. The async function returns a Promise.

```js
const f1 = () => 1;
const f2 = async () => 1;
assert(typeof f() === 'number');
assert(f2() instanceof Promise);
```

Reply:
I assumed r.pull() returns a Promise, in which case both should be identical.

Reply:
`pull` does not return a Promise. It is fire and forget. At one point (long time ago) it returned a promise that resolved when pull was done.